astropy_helpers/sphinx/ext/automodapi.py

```diff
@@ -52,6 +52,14 @@
         given, only objects that are actually in a subpackage of the package
         currently being documented are included.
 
+    * ``:inherited-members: True | False``
```

Comment:
The fact that it's a global configuration option shouldn't be documented here.  This part of the documentation is for options to the *directive*, not configuration settings.  You should instead add a ``automodsumm_inherited_members`` bullet to the list immediately after this (the one right after "This extension also adds two sphinx configuration options"), and the documentation here should just document  ``:inherited-members:`` (with no ``True | False``, as the True/False status is set by its presence or absence)